packages/drivers/routerlicious-driver/src/restWrapper.ts

```diff
@@ -102,11 +102,11 @@ export class RouterliciousRestWrapper extends RestWrapper {
             }, responseBody.retryAfter * 1000));
         }
 
+        const responseSummary = responseBody !== undefined
+            ? typeof responseBody === "string" ? responseBody : safeStringify(responseBody)
+            : response.statusText;
         throwR11sNetworkError(
-            "r11sFetchError",
-            responseBody !== undefined
-                ? typeof responseBody === "string" ? responseBody : safeStringify(responseBody)
-                : response.statusText,
+            `R11s fetch error: ${responseSummary}`,
```

Comment:
@znewton FYI on this change. I doubt you were using `fluidErrorCode` in telemetry but if you were, you can parse out the part of the message before the `:` to get a consistent message regardless of the `responseSummary` string which we don't control.